UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPartContainerBase.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using NaughtyAttributes;
+using UnityEngine;
+
+namespace HealthV2
+{
+	public enum BodyPartType
+	{
+		Head,
+		Chest,
+		LeftArm,
+		RightArm,
+		LeftLeg,
+		RightLeg,
+		Buttocks,
+		Implant
+	}
+
+	public class BodyPartContainerBase : MonoBehaviour
+	{
+		[SerializeField]
+		[Required("Need a health master to send updates too." +
+		          "Will attempt to find a components in its parents if not already set in editor.")]
+		private LivingHealthMasterBase healthMaster = null;
+
+		private ItemStorage storage;
```

Comment:
Codacy found an issue: [Remove the field 'storage' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6324510)